### PR TITLE
Enhanced many things

### PR DESCRIPTION
**Pull request by RDIL**
## New in this PR

* https://helmetjs.github.io/docs/dont-sniff-mimetype/
* Removed tag closings - slightly minifies HTML for faster delivery
* Synced GraphQL schema
* Made port config more consistent cross-platform
* npm did something with the lockfile, probably for the best

### CODE REVIEW

**Comment by RDIL** [comment]
@fkorotkov we should be good

File: package-lock.json [hunk 1]
```diff
@@ -12550,6 +12537,14 @@
...
       "dev": true
     },
+    "react-https-redirect": {
+      "version": "1.1.0",
+      "resolved": "https://registry.npmjs.org/react-https-redirect/-/react-https-redirect-1.1.0.tgz",
+      "integrity": "sha512-HWN/Ftgi31W2OAFVNo5BO1n8C9D2W0L+sque20dKuwh51hQuMwK4bMAtdrYAmapfGGh+53zQ1NsrSacVS1Q/mw==",
+      "requires": {
+        "prop-types": "^15.5.8"
+      }
+    }
```
Comment (RDIL) [inline package-lock.json]:
```suggestion
    },
```

**Comment by fkorotkov** [comment]
Can you still reproduce that http://cirrus-ci.com doen't redirect to the HTTPS version?

```
$: curl -I http://cirrus-ci.com
HTTP/1.1 200 OK
Last-Modified: Sat, 20 Jul 2019 14:23:40 GMT
Content-Length: 7133
Content-Disposition: inline; filename="index.html"
Accept-Ranges: bytes
Content-Type: text/html; charset=utf-8
Vary: Accept-Encoding
Cache-Control: public,max-age=3600
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
Date: Tue, 23 Jul 2019 18:03:33 GMT
Via: 1.1 google
```

It has `Strict-Transport-Security` header which should instruct your browser to use HTTPS only

**Comment by RDIL** [comment]
Interesting, I’ll try on my PC when I get home.

File: package-lock.json [hunk 2]
```diff
@@ -12550,6 +12537,14 @@
...
       "dev": true
     },
+    "react-https-redirect": {
+      "version": "1.1.0",
+      "resolved": "https://registry.npmjs.org/react-https-redirect/-/react-https-redirect-1.1.0.tgz",
+      "integrity": "sha512-HWN/Ftgi31W2OAFVNo5BO1n8C9D2W0L+sque20dKuwh51hQuMwK4bMAtdrYAmapfGGh+53zQ1NsrSacVS1Q/mw==",
+      "requires": {
+        "prop-types": "^15.5.8"
+      }
+    },
```
Comment (RDIL) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 3]
```diff
@@ -12550,6 +12537,14 @@
       "integrity": "sha512-X1Y+0jR47ImDVr54Ab6V9eGk0Hnu7fVWGeHQSOXHf/C2pF9c6uy3gef8QUeuUiWlNb0i08InPSE5a/KJzNzw1Q==",
       "dev": true
     },
+    "react-https-redirect": {
+      "version": "1.1.0",
+      "resolved": "https://registry.npmjs.org/react-https-redirect/-/react-https-redirect-1.1.0.tgz",
+      "integrity": "sha512-HWN/Ftgi31W2OAFVNo5BO1n8C9D2W0L+sque20dKuwh51hQuMwK4bMAtdrYAmapfGGh+53zQ1NsrSacVS1Q/mw==",
+      "requires": {
+        "prop-types": "^15.5.8"
+      }
```
Comment (RDIL) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 4]
```diff
@@ -12550,6 +12537,14 @@
       "integrity": "sha512-X1Y+0jR47ImDVr54Ab6V9eGk0Hnu7fVWGeHQSOXHf/C2pF9c6uy3gef8QUeuUiWlNb0i08InPSE5a/KJzNzw1Q==",
       "dev": true
     },
+    "react-https-redirect": {
+      "version": "1.1.0",
+      "resolved": "https://registry.npmjs.org/react-https-redirect/-/react-https-redirect-1.1.0.tgz",
+      "integrity": "sha512-HWN/Ftgi31W2OAFVNo5BO1n8C9D2W0L+sque20dKuwh51hQuMwK4bMAtdrYAmapfGGh+53zQ1NsrSacVS1Q/mw==",
+      "requires": {
+        "prop-types": "^15.5.8"
```
Comment (RDIL) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 5]
```diff
@@ -12550,6 +12537,14 @@
       "integrity": "sha512-X1Y+0jR47ImDVr54Ab6V9eGk0Hnu7fVWGeHQSOXHf/C2pF9c6uy3gef8QUeuUiWlNb0i08InPSE5a/KJzNzw1Q==",
       "dev": true
     },
+    "react-https-redirect": {
+      "version": "1.1.0",
+      "resolved": "https://registry.npmjs.org/react-https-redirect/-/react-https-redirect-1.1.0.tgz",
+      "integrity": "sha512-HWN/Ftgi31W2OAFVNo5BO1n8C9D2W0L+sque20dKuwh51hQuMwK4bMAtdrYAmapfGGh+53zQ1NsrSacVS1Q/mw==",
+      "requires": {
```
Comment (RDIL) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 6]
```diff
@@ -12550,6 +12537,14 @@
       "integrity": "sha512-X1Y+0jR47ImDVr54Ab6V9eGk0Hnu7fVWGeHQSOXHf/C2pF9c6uy3gef8QUeuUiWlNb0i08InPSE5a/KJzNzw1Q==",
       "dev": true
     },
+    "react-https-redirect": {
+      "version": "1.1.0",
+      "resolved": "https://registry.npmjs.org/react-https-redirect/-/react-https-redirect-1.1.0.tgz",
+      "integrity": "sha512-HWN/Ftgi31W2OAFVNo5BO1n8C9D2W0L+sque20dKuwh51hQuMwK4bMAtdrYAmapfGGh+53zQ1NsrSacVS1Q/mw==",
```
Comment (RDIL) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 7]
```diff
@@ -12550,6 +12537,14 @@
       "integrity": "sha512-X1Y+0jR47ImDVr54Ab6V9eGk0Hnu7fVWGeHQSOXHf/C2pF9c6uy3gef8QUeuUiWlNb0i08InPSE5a/KJzNzw1Q==",
       "dev": true
     },
+    "react-https-redirect": {
+      "version": "1.1.0",
+      "resolved": "https://registry.npmjs.org/react-https-redirect/-/react-https-redirect-1.1.0.tgz",
```
Comment (RDIL) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 8]
```diff
@@ -12550,6 +12537,14 @@
       "integrity": "sha512-X1Y+0jR47ImDVr54Ab6V9eGk0Hnu7fVWGeHQSOXHf/C2pF9c6uy3gef8QUeuUiWlNb0i08InPSE5a/KJzNzw1Q==",
       "dev": true
     },
+    "react-https-redirect": {
+      "version": "1.1.0",
```
Comment (RDIL) [inline package-lock.json]:
```suggestion
```

File: package-lock.json [hunk 9]
```diff
@@ -12550,6 +12537,14 @@
       "integrity": "sha512-X1Y+0jR47ImDVr54Ab6V9eGk0Hnu7fVWGeHQSOXHf/C2pF9c6uy3gef8QUeuUiWlNb0i08InPSE5a/KJzNzw1Q==",
       "dev": true
     },
+    "react-https-redirect": {
```
Comment (RDIL) [inline package-lock.json]:
```suggestion
```

**Comment by RDIL** [comment]
@fkorotkov I think this is ready to be shipped as a enhancement PR.

**Pull request merged**